CHANGELOG.md

```diff
@@ -6,9 +6,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.0.0] - 2019-02-21
+
 ### Added
 - The service broker will now automatically generate the org and space policy when 
   the service is provisioned into a CF space.
+- Added service broker environment parameter for `CONJUR_FOLLOWER_URL`. When set, the
+  service broker will provide the URL of a follower to an application for retrieving secret
+  values.
```

Comment:
There's new behavior on deprovision now too that we should describe. Also, can you please check the epic for specific CF component versions that are required? I'd prefer citing those rather than PCF versions here